modules/web/js/expression-editor/expression-editor-utils.js

```diff
@@ -49,9 +49,13 @@ class ExpressionEditor {
         this.ballerinaFileEditor = ballerinaFileEditor;
 
         // Get the expression for the statement or expression.
-        const expression = props.getterMethod instanceof Function ? props.getterMethod.call() :
+        let expression = props.getterMethod instanceof Function ? props.getterMethod.call() :
             (_.isNil(props.model.getSource()) ? '' : props.model.getSource(true).replace(/(?:\r\n|\r|\n)/g, ' '));
 
+        expression = expression.substr(expression.length - 1) === ';'
```

Comment:
We can use endsWith function here.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith